### PR TITLE
Remove the altitude filter

### PR DESCRIPTION
**Pull request by simoexpo**

### CODE REVIEW

File: src/libs/map-engine.ts [hunk 1]
```diff
@@ -163,32 +163,27 @@ export default class MapEngine {
...
+          longitude,
+          latitude,
+          { direction, enabled, marker: oldFlightInfo?.marker, zoom },
+        ).marker!;
+        google.maps.event.clearListeners(marker, 'click');
+        google.maps.event.addListener(marker, 'click', () => this.openPopupForFlight(flight, marker));
+        if (this.icaoNumberToPopup.icaoNumber === icaoNumber && this.icaoNumberToPopup.popup) {
+          this.icaoNumberToPopup.popup.setContent(createPopup(flight));
+        }
+        if (!oldFlightInfo) {
```
Comment (massimosiani) [inline src/libs/map-engine.ts]: do you see any drawback in removing this check? Probably the next two lines are idempotent.

Comment (simoexpo) [reply src/libs/map-engine.ts]: No drawback, I made a test without this check and it was good. The only reason I kept the check is related to the performance. I made a little performance test with and without the `if` and it seems that without this check the system load is a bit higher but... honestly I'm not sure we can trust the result :|

Comment (massimosiani) [reply src/libs/map-engine.ts]: Nice. Let's prefer performance if it's measurable.

Comment (simoexpo) [reply src/libs/map-engine.ts]: after some other tests I decided to remove the check

**Pull request merged**